instrumentation/http/src/main/java/brave/http/HttpClientHandler.java

```diff
@@ -125,7 +128,12 @@ public Span handleSend(HttpClientRequest request, Span span) {
     if (request == null) throw new NullPointerException("request == null");
     if (span == null) throw new NullPointerException("span == null");
     defaultInjector.inject(span.context(), request);
-    return handleStart(new HttpClientRequest.ToHttpAdapter(request), request.unwrap(), span);
+
+    Object unwrapped = request.unwrap();
+    if (unwrapped == null) unwrapped = NULL_SENTINEL; // Handle bad implementation
```

Comment:
Worth doing a log-once for these sort of bad implementation handlers? I always wonder how a bad implementation will become a good one if we just handle it silently.